### ISSUE TITLE
Adjust Spaces to match requested copy

### ISSUE THREAD

**Issue by jeremyf** [forem/forem#17097]
Using the following [figma design](https://www.figma.com/file/SeTWMa3E7F7fieVKm2wvCt/Roles-and-Authorization-System?node-id=309%3A3716) adjust 

- Campaigns = https://admin.forem.com/docs/advanced-customization/config/campaigns
- Organizations = https://admin.forem.com/docs/managing-your-community/organization-pages#overview.
- Replace "Only admins can publish posts. Other members can only react to and comment on posts. Only admins will be able to post on behalf of organizations." with "Only admins can publish posts. Other members can only react to and comment on posts.  For organizations, only admins who are part of organization will be able to post on behalf of that organization. [Learn more](https://admin.forem.com/docs/managing-your-community/organization-pages#overview)

Related to forem/forem#17032

**Comment by amywtlin** [comment]
@ludwiczakpawel Can you let Jeremy know where he can get that (i) icon? I think we should have used it in our application, right?

**Comment by ludwiczakpawel** [comment]
@jeremyf  we have a [method](https://storybook.forem.com/?path=/docs/components-icons--default#rails) to call all the SVGs from `/app/assets/images/` where we also have one for that _info_ icon.

```
crayons_icon_tag("info")
```

^ this one is actually outlined so slightly different than what was used in the design. i'd say it's fine to use this one OR you can an icon by yourself as well and call it `info-filled.svg`.

On that same storybook docs page, [there's a section describing how to do it](https://storybook.forem.com/?path=/docs/components-icons--default#rules) and how to optimize SVGs (just FYI) but below is already optimize code for you:

```
<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M2 12c0 5.523 4.477 10 10 10s10-4.477 10-10S17.523 2 12 2 2 6.477 2 12Zm9-3V7h2v2h-2Zm0 8v-6h2v6h-2Z" /></svg>
```

**Comment by jeremyf** [comment]
tl;dr - Designer says "it's slightly different but okay to use" --> developer hears "ship it"
